assets/css/frontend/base/wordpress.css

```diff
@@ -0,0 +1,31 @@
+/* WordPress generated classes */
+
+.alignleft {
+	float: left;
+	margin-right: 1rem;
```

Comment:
Should we add text align rules to both alignleft and alignright (as we are doing with aligncenter)?

Definitely more of a general question, I've always kind of just done that myself, but not sure if it's necessary.

Thanks for this PR by the way @smy315 😄

Reply:
@magnificode, text-align left isn't needed since it's the default value and line 28 covers .alignright for wp-captions. I did some testing, and text-align for .alignleft (line 3) and .alignright (line 8) is not needed. I might be wrong but I don't think it's necessary.

Reply:
@smy315 I can definately see the need for a text-align: left if the component is aligned center by default. Let's add that in there.